src/runtime/entries/stormkit.ts

```diff
@@ -34,6 +34,7 @@ export const handler: Handler<StormkitEvent, StormkitResult> = async function (
     body: event.body,
   });
 
+  // @todo handle cookies
```

Comment:
We probably can remove this todos. `normalizeOutgoingHeaders` should be able to fix this

Reply:
I believe stormkit just wraps lambda so the cookies needs to be handled like in the `aws-lambda` preset

Reply:
This needs to be done in netlify-lambda too

Reply:
Sure if it is an easy thing to do we can make this PR a fix to normalize cookies for all presets...